data/lib/src/network/linshare_http_client.dart

```diff
@@ -128,6 +131,25 @@ class LinShareHttpClient {
     return resultJson.map((data) => _getDynamicAutoCompleteResult(data)).toList();
   }
 
+  Future<List<WorkGroupNodeDto>> getWorkGroupChildNodes(
+    SharedSpaceId sharedSpaceId, [
+    WorkGroupNodeId parentId,
+  ]) async {
+    final endpointPath = Endpoint.sharedSpaces
+        .withPathParameter('/${sharedSpaceId.uuid}')
+        .withPathParameter(Endpoint.nodes)
+        .generateEndpointPath();
+
+    final queryParameters = parentId != null ? {'parent': parentId.uuid} : null;
```

Comment:
I created a extension function to convert from our query list to map, please consider to refactor to this.
`final queryParameters = parentId != null ? [StringQueryParameter('parent', parentId.uuid)].toMap() : {}`